jobs/config.json

```diff
@@ -6905,7 +6905,7 @@
       "--provider=gke",
       "--test_args=--ginkgo.skip=\\[Flaky\\]|\\[Feature:.+\\] --kubectl-path=../../../../kubernetes_skew/cluster/kubectl.sh --minStartupPods=8",
       "--timeout=900m",
-      "--upgrade_args=--ginkgo.focus=\\[Feature:ClusterUpgrade\\] --upgrade-target=ci/latest --upgrade-image=gci"
+      "--upgrade_args=--ginkgo.focus=\\[Feature:ClusterUpgrade\\] --upgrade-target=gs://kubernetes-release-gke/release/v1.8.0-gke.1 --upgrade-image=gci"
```

Comment:
I think you want just `v1.8.0-gke.1`, since this is used by the e2e tests, not kubetest.

Reply:
done, that's smarter than kubetest